server/test/services/applicant/question/EnumeratorQuestionTest.java

```diff
@@ -122,6 +136,50 @@ public void withDuplicateNames_hasValidationErrors() {
                         MessageKey.ENUMERATOR_VALIDATION_DUPLICATE_ENTITY_NAME))));
   }
 
```

Comment:
**suggestion:** Could we add one more test that it passes validation when they set a min / max and the number of items a person adds is within that?

Reply:
Good idea - added!